### PR TITLE
T01

### PR DESCRIPTION
**Pull request by dineshN52**
Number conversion game updated

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
-        }
-    }
+﻿using System;
+namespace Numconv {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.WriteLine ("Enter the decimal number:");
+         int deci = int.Parse (Console.ReadLine ());
+         string hex = Convert.ToString (deci, 16); // Convert to hexadecimal
+         string binary = Convert.ToString (deci, 2);// Convert to binary
```
Comment (ravin-imm) [inline Program.cs:8]: Can you implement this without using the above methods?

**Review by ravin-imm — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+         Console.WriteLine ($"Integer {intValue} in hexadecimal: 0x{hexValue}");
+         string BiValueFromInt = IntToBi (intValue);
+         Console.WriteLine ($"Integer {intValue} as Binary: {BiValueFromInt}");
+         Console.ReadKey ();
+      }
+      static string IntToHex (int value) {
+         string hex = "";
+         while (value > 0) {
+            int remainder = value % 16;
+            hex = GetHexDigit (remainder) + hex;
```
Comment (ravin-imm) [inline Program.cs:17]: This can be simplified further as:
**_hex += GetHexDigit (value % 16);_**
Please make the necessary such changes wherever required.

**Review by ravin-imm — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 2 above)
Comment (dineshN52) [reply Program.cs]: Operator changed and updated sir

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,37 @@
...
+         Console.WriteLine ($"Integer {intValue} in hexadecimal: 0x{hexValue}");
+         string BiValueFromInt = IntToBi (intValue);
+         Console.WriteLine ($"Integer {intValue} as Binary: {BiValueFromInt}");
+         Console.ReadKey ();
+      }
+      static string IntToHex (int value) {
+         string hex = "";
+         while (value > 0) {
+            int remainder = value % 16;
+            hex += GetHexDigit (remainder);
```
Comment (ravin-imm) [inline Program.cs:17]: This can be simplified further as _remainder_ variable is not used further:
_**hex += GetHexDigit (value % 16)**_

**Review by ravin-imm — CHANGES_REQUESTED** [review]

**Review by ravin-imm — APPROVED** [review]

**Pull request closed without merging**